libpod/container_api.go

```diff
@@ -136,6 +137,61 @@ func (c *Container) Update(resources *spec.LinuxResources, restartPolicy *string
 	return c.update(resources, restartPolicy, restartRetries)
 }
 
+// UpdateHealthCheckConfig updates HealthCheck configuration the given container.
+func (c *Container) UpdateHealthCheckConfig(healthCheckConfig *manifest.Schema2HealthConfig, changedTimer bool, noHealthCheck bool) error {
```

Comment:
noHealthCheck can probably be `healthCheckConfig == nil`

Reply:
And I think we might want to compute `changedTimer` in here, instead of requiring the user pass it in - moving the rest of the parsing out of libpod was good but this was maybe a step too far.